reportssql/indicator_functions.sql

```diff
@@ -913,6 +913,39 @@ WHERE
 END$$ 
 DELIMITER ;
 
+DROP FUNCTION IF EXISTS Testing_Indicator3a;
+
+DELIMITER $$
+CREATE FUNCTION Testing_Indicator3a(
+    p_startDate DATE,
+    p_endDate DATE,
+    p_startAgeInMonths INT(11),
+    p_endAgeInMonths INT (11),
+    p_includeStartAge TINYINT(1)) RETURNS INT(11)
+    DETERMINISTIC
+BEGIN
+    DECLARE result INT(11) DEFAULT 0;
+
+SELECT
+    COUNT(DISTINCT pat.patient_id) INTO result
+FROM
+    patient pat
+WHERE
+    patientAgeAtReportEndDateIsBetween(pat.patient_id, p_endDate, p_startAgeInMonths, p_endAgeInMonths) AND
+    patientHadAVirologicHIVTestDuringReportingPeriod(pat.patient_id, p_startDate, p_endDate) AND
+    patientHadAPositiveVirologicHIVTestResultDuringReportingPeriod(pat.patient_id, p_startDate, p_endDate) AND
+    patientMostRecentVirologicHIVTestResultIsPositive(pat.patient_id) AND
+    patientWasNotEnrolledToHIVProgramBeforeVirologicTest(pat.patient_id, p_startDate, p_endDate) AND
```

Comment:
@tmvumbi2 Can't we remove this function and use:

NOT patientHasEnrolledIntoHivProgramBefore(pat.patient_id,dateOfVirologicHIVTest) AND